packages/smooth_app/lib/pages/product/edit_product_page.dart

```diff
@@ -193,22 +183,27 @@ class _EditProductPageState extends State<EditProductPage> {
     );
   }
 
-  Widget _getSimpleListTileItem(final AbstractSimpleInputPageHelper helper) =>
-      _ListTitleItem(
-        title: helper.getTitle(),
-        subtitle: helper.getSubtitle(),
-        onTap: () async {
-          if (!await ProductRefresher().checkIfLoggedIn(context)) {
-            return;
-          }
-          await Navigator.push<Product>(
-            context,
-            MaterialPageRoute<Product>(
-              builder: (BuildContext context) => SimpleInputPage(helper),
+  Widget _getSimpleListTileItem(final AbstractSimpleInputPageHelper helper) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+    return _ListTitleItem(
+      title: helper.getTitle(appLocalizations),
+      subtitle: helper.getSubtitle(appLocalizations),
+      onTap: () async {
+        if (!await ProductRefresher().checkIfLoggedIn(context)) {
+          return;
+        }
```

Comment:
Please add a todo that we should show something there, maybe a snackbar

Reply:
Thank you @M123-dev!
Actually there's already a dialog in `checkIfLoggedIn`.